slides/intro-napari.qmd

```diff
@@ -0,0 +1,213 @@
+## Opening Napari
+
+```{.bash}
+napari
+```
+
+![](img/blank-napari-ui.png){alt="A screenshot of the default Napari user 
+interface"}
+
+## Opening images
+
+`File > Open Files(s)`, then navigate to `calcium imaging folder` and open `translation1_00001_ce.tif`
```

Comment:
Is this going to be on the shared drive?

Reply:
yep